pkgs/development/tools/misc/mkrom/default.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchFromGitHub, stdenv, cmake, asciidoc, libxslt }:
+
+
+stdenv.mkDerivation rec {
+  pname = "mkrom";
+
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "KnightOS";
+    repo = "mkrom";
+    rev = version;
+    sha256 = "0smilzkpf4v4w5lbx4jaah8hr24110makxjy3j92qb404pxcb5r1";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ asciidoc libxslt.bin ];
```

Comment:
These should be in `nativeBuildInputs`. Also, is `libxslt.bin` necessary? `asciidoc` should already have the correct path to `xsltproc`.

Reply:
Yes, `libxslt.bin` seems to be necessary.